knowledge-graph/src/main/scala/io/renku/knowledgegraph/datasets/DatasetSearchResult.scala

```diff
@@ -32,48 +32,48 @@ import io.renku.tinytypes.constraints.NonNegativeInt
 import io.renku.tinytypes.{IntTinyType, TinyTypeFactory}
 
 final case class DatasetSearchResult(
-    id:                  Identifier,
-    title:               Title,
-    name:                Name,
-    maybeDescription:    Option[Description],
-    creators:            List[DatasetCreator],
-    date:                Date,
-    exemplarProjectPath: projects.Path,
-    projectsCount:       ProjectsCount,
-    keywords:            List[Keyword],
-    images:              List[ImageUri]
+    id:               Identifier,
+    title:            Title,
+    name:             Name,
+    maybeDescription: Option[Description],
+    creators:         List[DatasetCreator],
+    date:             Date,
+    exemplarProject:  ExemplarProject,
+    projectsCount:    ProjectsCount,
+    keywords:         List[Keyword],
+    images:           List[ImageUri]
 )
 
 object DatasetSearchResult {
 
-  implicit def encoder(implicit
-      gitLabUrl:   GitLabUrl,
-      renkuApiUrl: config.renku.ApiUrl
-  ): Encoder[DatasetSearchResult] = Encoder.instance[DatasetSearchResult] {
-    case DatasetSearchResult(id,
-                             title,
-                             name,
-                             maybeDescription,
-                             creators,
-                             date,
-                             exemplarProjectPath,
-                             projectsCount,
-                             keywords,
-                             images
-        ) =>
-      json"""{
+  final case class ExemplarProject(id: projects.ResourceId, path: projects.Path)
```

Comment:
For curiosity: why is it called "Exemplar" - it sounds a bit like being test data :) Or is this the example for the api docs?